bulwark/checks.py

```diff
@@ -60,32 +60,68 @@ def has_columns(df, columns, exact_cols=False, exact_order=False):
     return df
 
 
-def has_no_nans(df, columns=None):
-    """Asserts that there are no np.nans in `df`.
+def has_no_x(df, values=None, columns=None):
+    """Asserts that there are no user specified values in `df`.
```

Comment:
Asserts that there are no user-specified `values` in `df`'s `columns`.

Reply:
Done!